libs/core-ui/src/lib/Interfaces/IDataset.ts

```diff
@@ -11,13 +11,14 @@ export enum DatasetTaskType {
   TextClassification = "text_classification",
   MultilabelTextClassification = "multilabel_text_classification",
   MultilabelImageClassification = "multilabel_image_classification",
-  Forecasting = "forecasting"
+  Forecasting = "forecasting",
+  ObjectDetection = "object_detection"
 }
 
 export interface IDataset {
   task_type: DatasetTaskType;
-  true_y: number[] | number[][];
-  predicted_y?: number[] | number[][];
+  true_y: number[] | number[][] | number[][][];
```

Comment:
I think there might be some test failures related to this change which may require other file changes - also, did you converge on using this format vs the additional object detection variable?